test/unit/test_formula.py

```diff
@@ -38,17 +42,45 @@ def test_generate_formula():
         # - punctuation
         # - trailing whitespace
         # - extra capitilization
-        'description': 'A tool to release... scripts, binaries, and executables to GitHub. ',
```

Comment:
Please leave the description as it was. If you look at the comment in the code directly above it, it explains why it was written the way it was:

>         # We use a badly written description string here on purpose to test our formatting code, this includes...